Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -732,50 +844,54 @@ def get_team_aad_id(team_name: str) -> str:
 
 
 def get_user(user: str) -> list:
-    """Retrieves the AAD ID of requested user
+    """Retrieves the AAD ID of requested user and the userType
 
     Args:
         user (str): Display name/mail/UPN of user to get ID of.
 
     Return:
         list: List containing the requsted user object
     """
+    demisto.debug(f"Given user = {user}")
     url: str = f'{GRAPH_BASE_URL}/v1.0/users'
     params = {
         '$filter': f"displayName eq '{user}' or mail eq '{user}' or userPrincipalName eq '{user}'",
-        '$select': 'id'
+        '$select': 'id, userType'
     }
     users = cast(Dict[Any, Any], http_request('GET', url, params=params))
     return users.get('value', [])
 
 
-def add_user_to_channel(team_aad_id: str, channel_id: str, user_id: str):
+def add_user_to_channel(team_aad_id: str, channel_id: str, user_id: str, is_owner: bool = False):
     """
     Request for adding user to channel
     """
-    url: str = f'{GRAPH_BASE_URL}/beta/teams/{team_aad_id}/channels/{channel_id}/members'
-    requestjson_: dict = {
-        '@odata.type': '#microsoft.graph.aadUserConversationMember',
-        'roles': [],
-        'user@odata.bind': f'https://graph.microsoft.com/beta/users/{user_id}'  # disable-secrets-detection
-    }
-    http_request('POST', url, json_=requestjson_)
+    url = f'{GRAPH_BASE_URL}/v1.0/teams/{team_aad_id}/channels/{channel_id}/members'
```

Comment:
good job!
this would solve the issue for this [bug](https://jira-hq.paloaltonetworks.local/browse/XSUP-21062) 
@amshamah419 FYI